lib/rustler_precompiled/config.ex

```diff
@@ -52,6 +69,16 @@ defmodule RustlerPrecompiled.Config do
     end
   end
 
+  defp validate_targets!(nil), do: raise_for_nil_field_value(:targets)
+
+  defp validate_targets!(targets) do
+    if is_list(targets) do
```

Comment:
It would be nice if we had a list of all available targets and check upon that list (from Rust plataform support page). I think we can do that in another PR.

Reply:
Absolutely. That would be useful!